### ISSUE TITLE
shuld we make logs more friendly about tsdb query.

### ISSUE THREAD

**Issue by lynchlee** [OpenTSDB/opentsdb#675]
while i make a query with　/api/query?start=1452679199&m=sum:1d-sum:application.collection.users{cluster=localhost,appkey=*}, i got logs :
2016-01-15 18:20:41,877 INFO  [New I/O worker #10] n.o.c.TsdbQuery#call[line:395]: TsdbQuery(start_time=1452679199000, end_time=1452853241869, metric=[0, 0, 3], tags=[[0, 0, 2, 0, 0, 9]], rate=false, aggregator=sum, group_bys=([0, 0, 6], )) matched 2 rows in 1 spans in 10ms

e..... should we log them like this:
2016-01-15 18:20:41,877 INFO  [New I/O worker #10] n.o.c.TsdbQuery#call[line:395]: TsdbQuery(start_time=1452679199000, end_time=1452853241869, metric=[some string], tags=[some string], rate=false, aggregator=sum, group_bys=([some string], )) matched 2 rows in 1 spans in 10ms

**Comment by manolama** [comment]
Yeah we definitely need to fix up that logging. I've actually been working on this lately and have a fully JSONified output that's Splunk friendly. Just need to upstream it. E.g.:

'''
{
	"query": {
		"start": "1h-ago",
		"end": null,
		"timezone": null,
		"options": null,
		"padding": false,
		"queries": [{
			"aggregator": "zimsum",
			"metric": "tsd.http.query.success.sized_bucket.latency.count",
			"tsuids": null,
			"downsample": null,
			"rate": true,
			"filters": [{
				"tagk": "host",
				"filter": "myhost",
				"group_by": false,
				"type": "literal_or"
			}, {
				"tagk": "dps",
				"filter": "fetched",
				"group_by": true,
				"type": "literal_or"
			}],
			"fuzzy": false,
			"rateOptions": null,
			"explicitTags": false,
			"preAggregate": false,
			"rollupUsage": null,
			"rollupTable": "raw",
			"tags": {
				"dps": "literal_or(fetched)"
			}
		}],
		"timeout": 0,
		"noAnnotations": false,
		"globalAnnotations": false,
		"showTSUIDs": true,
		"showStats": false,
		"showSummary": true,
		"msResolution": false
	},
	"exception": null,
	"executed": 1,
	"user": "clarsen",
	"remote": "ip",
	"requestHeaders": {
		"Origin": "chrome-extension://",
		"Cookie": "*******",
		"Accept": "*/*",
		"User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.111 Safari/537.36",
		"X-Forwarded-Host": "hostname",
		"Connection": "close",
		"Host": "localhost",
	},
	"numRunningQueries": 0,
	"v2Query": null,
	"queryStartTimestamp": 1454301943234,
	"queryCompletedTimestamp": 1454301944833,
	"stats": {
		"avgAggregationTime": 41.512508,
		"avgHBaseTime": 367.378805,
		"avgQueryScanTime": 1156.467453,
		"avgSaltScannerTime": 372.212276,
		"avgScannerUidToStringTime": 0.0,
		"bytesFromStorage": 2744126,
		"columnsFromStorage": 38440,
		"dpsPostFilter": 38440,
		"emittedDPs": 141,
		"maxAggregationTime": 41.512508,
		"maxHBaseTime": 1129.862789,
		"maxQueryScanTime": 1156.467453,
		"maxScannerUidtoStringTime": 0.0,
		"maxUidToStringTime": 14.240267,
		"nanDPs": 0,
		"queryIdx_00": {
			"aggregationTime": 41.512508,
			"avgHBaseTime": 367.378805,
			"avgSaltScannerTime": 372.212276,
			"avgScannerUidToStringTime": 0.0,
			"bytesFromStorage": 2744126,
			"columnsFromStorage": 38440,
			"dpsPostFilter": 38440,
			"emittedDPs": 141,
			"groupByTime": 10.456058,
			"maxHBaseTime": 1129.862789,
			"maxScannerUidtoStringTime": 0.0,
			"nanDPs": 0,
			"queryIndex": 0,
			"queryScanTime": 1156.467453,
			"rowsFromStorage": 726,
			"rowsPostFilter": 726,
			"saltScannerMergeTime": 18.335938,
			"stringToUidTime": 319.284333,
			"successfulScan": 20,
			"uidPairsResolved": 0,
			"uidToStringTime": 14.240267
		},
		"rowsFromStorage": 726,
		"rowsPostFilter": 726,
		"serializationTime": 68.995988,
		"successfulScan": 20,
		"totalTime": 1598.236821,
		"uidPairsResolved": 0
	},
	"httpResponse": {
		"code": 200,
		"reasonPhrase": "OK"
	}
}
'''

**Comment by bobrik** [comment]
@manolama :+1:

**Comment by lynchlee** [comment]
@manolama :+1: OK. got it.

**Comment by johann8384** [comment]
I'll bring up the discussion on the mailing list. I can go over all the current logging for consistency and clarity. Depending on what @manolama has done, we can probably just use a Logback encoder to change between "plain text" and JSON output.

**Comment by johann8384** [comment]
@manolama any chance you'd like to upstream this? :)
